projects/plugins/boost/app/assets/src/js/pages/getting-started/GettingStarted.svelte

```diff
@@ -0,0 +1,229 @@
+<script>
+	import { derived } from 'svelte/store';
+	import { __ } from '@wordpress/i18n';
+	import ContextInfo from '../../elements/ContextInfo.svelte';
+	import ProductPrice from '../../elements/ProductPrice.svelte';
+	import TemplatedString from '../../elements/TemplatedString.svelte';
+	import Header from '../../sections/Header.svelte';
+	import { getStarted } from '../../stores/config';
+	import config from '../../stores/config';
+	import externalLinkTemplateVar from '../../utils/external-link-template-var';
+	import { getUpgradeURL } from '../../utils/upgrade';
+
+	const pricing = derived( config, $config => $config.pricing );
```

Comment:
Something @pyronaur taught me: this can be done like so:
```
$: pricing = $config.pricing;
```

It's easier to read at a glance, and is equally good at keeping `pricing` up to date as `$config` changes, due to `$:`. You can even skip the `let` as Svelte implies that for you. :)